src/platform/plugins/shared/saved_objects_management/common/constants/ml_saved_object_types.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+export const ML_SAVED_OBJECT_TYPES = new Set(['ml-job', 'ml-trained-model']);
```

Comment:
I wonder if we should move the constants declared [here](https://github.com/elastic/kibana/blob/0c106d331b3c1b946c4c706cf6afafee0c8377c4/x-pack/platform/plugins/shared/ml/common/types/saved_objects.ts#L14) to a package and include this Set.

In this PR `ml-job` and `ml-trained-model` are used a few times and it would be safer if they all came from the same place.